twitchobserver/twitchobserver.py

```diff
@@ -216,13 +250,114 @@ def unban_user(self, user, channel):
 
         self.send_message("/unban {}".format(user), channel)
 
-    def clear_chat_history(self, channel):
-        """Clears the chat history of a channel.
+    def ignore_user(self, nickname, channel):
+        """Ignores a particular user by not showing his messages in chat.
 
+        :param nickname: The nickname of the user
         :param channel: The channel name
         """
 
-        self.send_message("/clear", channel)
+        self.send_message("/ignore {}".format(nickname), channel)
+
+    def unignore_user(self, nickname, channel):
+        """Unignores a particular user by showing his messages in chat again.
+
+        :param nickname: The nickname of the user
+        :param channel: The channel name
+        """
+
+        self.send_message("/unignore {}".format(nickname), channel)
+
+    def timout_user(self, nickname, channel, duration=600):
+        """Timeouts a user by deleting all previously send messages and letting him not be able to send messages for ten minutes (default) or for `duration` of seconds.
+
+        :param nickname: The nickname of the user
+        :param channel: The channel name
+        :param duration: The duration to ban the user from chat (in seconds). Defaults to ten minutes
+        """
+
+        self.send_message("/timeout {} {}".format(nickname, duration), channel)
+
+    def set_slow_mode(self, channel, duration):
```

Comment:
Because it more closely mirrors the command. It lays the focus on the action. You directly now what the method will do. I like that.
But your version is more concise because it needs only one method to do the job. I can change it, no problem. :)